### PR TITLE
Ozone Bid Adapter: Ozone v2.6.0 adapter updates

### PR DESCRIPTION
**Pull request by afsheenb**
Type of change
• Bugfix
• Feature

Description of changes

• Added CCPA/US Privacy Support
• Added support for additional QueryParams
• Added support for Price Floors Module per #6465
• Added support for First Party Data Enrichment Module
• Bug Fix for ID5 per #6324
• Bug Fix for ensuring adId is unique to ozone by appending an -oz- prefix in the adId to solve feedback per #6381
• Bug Fix to support meta.advertiserDomains per #6466

    engineering@ozoneproject.com

### CODE REVIEW

**Comment by patmmccann** [comment]
@afsheenb thank you so much for this PR!

**Review by gwhigs — APPROVED** [review]
LGTM

**Comment by ChrisHuie** [comment]
@patmmccann @afsheenb @gwhigs Hey everyone. I had to revert this pr out because it seems too have a weird side effect on our testing. Looks like tests are passing but only initially connects and then quits out before testing suite is run. Can you look at this again please 🙏

File: test/spec/modules/ozoneBidAdapter_spec.js [hunk 1]
```diff
@@ -2464,6 +2656,20 @@ describe('ozone Adapter', function () {
...
       expect(result[0].url).to.include('gdpr_consent=BOh7mtYOh7mtYAcABBENCU-AAAAncgPIXJiiAoao0PxBFkgCAC8ACIAAQAQQAAIAAAIAAAhBGAAAQAQAEQgAAAAAAABAAAAAAAAAAAAAAACAAAAAAAACgAAAAABAAAAQAAAAAAA');
     });
+    it('should append ccpa (usp data)', function() {
+      // get the cookie bag populated
+      spec.buildRequests(validBidRequests, validBidderRequest.bidderRequest);
+      const result = spec.getUserSyncs({iframeEnabled: true}, 'good server response', gdpr1, '1YYN');
+      expect(result).to.be.an('array');
+      expect(result[0].url).to.include('usp_consent=1YYN');
+    });
+    it.only('should use "" if no usp is sent to cookieSync', function() {
```
Comment (patmmccann) [inline test/spec/modules/ozoneBidAdapter_spec.js:2666]: Could this be an issue,  I don't see this synyax used elsewhere @ChrisHuie

**Comment by patmmccann** [comment]
@askrupert-dm as fyi as well

If you guys notice for each browser circle ci only ran one test but reported succeeded.  Could you guys resubmit?

**Pull request merged**